article.md

```diff
@@ -0,0 +1,326 @@
+# How to Build a LinkedIn Job Posting Scraper using Crawlee for Python and Streamlit
+
+
+## Introduction
+
+In this article, we are going to be building a web application to scrape LinkedIn’s website for job postings using Crawlee and Streamlit.
+
+We will create a LinkedIn job scraper utilising Crawlee for Python to extract the company name, job title, time of posting and link to the job posting, from dynamically received user input through the web application.
+
+By the end of this tutorial, you’ll have had a fully functional web application that you can use to scrape job postings from the LinkedIn website.
+
+
+
+![alt_text](images/image1.png "image_tooltip")
+
+
+Let’s get started
+
+
+## Prerequisites
+
+Let's start by installing Crawlee for Python with this command:
+
+
+```
+pipx run crawlee create linkedin-scraper
+```
+
+
+Select `PlaywrightCrawler` in the terminal when Crawlee asks for it.
+
+After installation, Crawlee for Python will create a boilerplate code for you. Redirect into the project folder and run this command to install dependencies.
+
+
+```
+Poetry install
+```
+
+
+
+## Building the LinkedIn Scraper using Crawlee for Python
+
+In this section, we are going to be building the scraper using the Crawlee Python package. To learn more about Crawlee, check out their [documentation](https://crawlee.dev/python/docs/quick-start).
+
+				
+
+
+### Inspecting the LinkedIn job Search Page
+
+Open LinkedIn on your web browser and sign out from the website (if you already have an account logged in). You should see an interface like this.
+
+
+
+
+![alt_text](images/image2.png "image_tooltip")
+
+
+Navigate to the jobs section, search for a job and location of your choice; copy the Uniform Resource Locator (URL).
+
+
+
+![alt_text](images/image3.png "image_tooltip")
+
+
+You should have something like this:
+
+`https://www.linkedin.com/jobs/search?keywords=Backend%20Developer&location=Canada&geoId=101174742&trk=public_jobs_jobs-search-bar_search-submit&position=1&pageNum=0` 
+
+The URL can be divided into two:
+
+
+
+* The base URL
+* The parameters
+
+The base URL is usually before the question mark while the parameters come after the question mark. Furthermore, you can change the inputs to the parameters to get different results on your webpage.
```

Comment:
I don't think this is correct terminology. I suggest rephrasing this so that it just says something like "We're going to focus on the search parameters, which is the part that goes after '?'. The keyword and location parameters are the most important ones for us".

I'd skip the explanation of base url/parameters completely.